.rubocop.yml

```diff
@@ -4,7 +4,3 @@ inherit_gem:
 
 Style/ClassVars:
   Enabled: false
-
-AllCops:
-  Exclude:
-    - 'vendor/**/*'
```

Comment:
Why has this been removed? Is it being automatically excluded by rubocop like `/tmp` is?

Reply:
It just appears to be unnecessary, as it's not affecting local tests or the ones here on CI. The fewer bits of config/overrides we have, the better.

Would you prefer it in a separate commit/PR?

Reply:
There's nothing else going on in this commit, so that's fine, but can you update the commit message?

Reply:
Sure, updated - does that look OK?